doc/classes/LookAtModifier3D.xml

```diff
@@ -120,9 +120,9 @@
 			The transition type of the time-based interpolation. See also [enum Tween.TransitionType].
 		</member>
 		<member name="use_angle_limitation" type="bool" setter="set_use_angle_limitation" getter="is_using_angle_limitation" default="false">
-			If [code]true[/code], limits the degree of rotation. This helps prevent the character's neck from rotating 360 degrees.
+			If [code]true[/code], limits the amount of rotation. For example, this helps to prevent a character's neck from rotating 360 degrees.
 			[b]Note:[/b] As with [AnimationTree] blending, interpolation is provided that favors [method Skeleton3D.get_bone_rest]. This means that interpolation does not select the shortest path in some cases.
-			[b]Note:[/b] Some [member transition_type] may exceed the limitations (e.g. `Back`, `Elastic`, and `Spring`). If interpolation occurs while overshooting the limitations, the result might possibly not respect the bone rest.
+			[b]Note:[/b] Some values for [member transition_type] (such as [constant Tween.TRANS_BACK], [constant Tween.TRANS_ELASTIC], and [constant Tween.TRANS_SPRING]) may exceed the limitations. If interpolation occurs while overshooting the limitations, the result might not respect the bone rest.
```

Comment:
I am genuinely not sure why this is noted here and not in `transition_type` itself. I also don't quite understand what the last sentence is trying to say.